pithos/pithos.py

```diff
@@ -22,13 +22,24 @@
 import signal
 
 import gi
-gi.require_version('Gst', '1.0')
 from gi.repository import Gst, GstPbutils, GObject, Gtk, Gdk, Pango, GdkPixbuf, Gio, GLib
 import contextlib
 import html
 import math
 import urllib.request, urllib.error, urllib.parse
 import json
+
+try:
+    gi.require_version('Gst', '1.0')
+except ValueError, e:
+    if e[0].startswith("Namespace Gst not available for version"):
+        print "Received error %s from Gstreamer." % e[0]
```

Comment:
Originally I was using the Pithos available for Wheezy but it wouldn't 
continue playing on some songs and wasn't sure if it was related to 
fresh purchase of Pandora One, so decided to build from source. 
Following the natural path of cloning and python setup.py build, I built 
and installed it without any issues (using default python on setup.py). 
Tried to launch newly built pithos (/usr/local/bin) and received gi 
version traceback (see also Issue 32). After adding a nicer version 
check (above) for gstreamer, another import failure on html; pip 
installed that. (A list of external python libraries/deps might be 
useful on the wiki.) Finally, the abort on urllib imports only in Python 
3 clued me in that it wasn't a Python2/3 app and realized further time 
investment was not useful. App looks good and useful but apparently not 
for me :)